docs/application/dotnet/guides/nui/nui-components/CameraView.md

```diff
@@ -0,0 +1,62 @@
+---
+keyword: camera, CameraView, NUI, CameraView.DisplayType
+---
+
+# CameraView
+
+The [CameraView](https://samsung.github.io/TizenFX/API9/api/Tizen.NUI.BaseComponents.CameraView.html) class is the NUI view that displays [camera](../../multimedia/camera.md).

```

Comment:
@JoogabYun, recently the  TizenFX API Reference links were updated for all pages
https://samsung.github.io/TizenFX -> /application/dotnet/api/TizenFX

see this PR
https://github.com/Samsung/tizen-docs/pull/1387

So this link should be changed to the /application/dotnet/api/TizenFX